content/_index.md

```diff
@@ -5,7 +5,7 @@ layout: landing
 
 carousel:
   - pre: Tools
-    title: Real-time Indicators of COVID-19 Activity
+    title: Real-time Indicators of Infectious Disease Activity
     ref: covidcast
     alt: Explore the COVIDcast Dashboard
```

Comment:
@rlunde do we still want to refer to the dashboard as "COVIDcast" here?  If we dont change this reference now, we will obviously do it when the dashboard is rebranded... 
 But with the intent of reducing COVID-specific text on entrypoints to the site, is it a too much of a "scare word" in this instance?

Reply:
I think we should probably refer to it as COVIDcast until we're ready to change all instances to "Delphi Epidata"?